ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -223,40 +221,37 @@ private class JdbcLedgerDao(
       }
     }
 
+  private val nonLocalParticipantId =
+    Ref.ParticipantId.assertFromString("__HACK__non-local-participant-id")
```

Comment:
HACK is a scary word, and probably to harsh, I would suggest saying RESTRICTED_NON_LOCAL_PARTICIPANT_ID. Effectively this is now a restricted string.

Should we maybe move it to the package object?

Reply:
I am fine with the placement and with the name, but also fine with restricted. What do you think @meiersi-da , @mziolekda ?

Reply:
Let's use @mziolekda 's proposal.